setup.py

```diff
@@ -26,5 +26,6 @@
     install_requires=[
         'Flask>=0.10',
         'six==1.9.0',
+        'requests==2.7.0',
```

Comment:
Is there a reason why we are doing imports differently in this repository than in all the others?  In the others we have `requirements.txt` rather than a `setup.py`.  I'm not against this, but it would be nice to do things the same way in all apps for consistency.